.travis.yml

```diff
@@ -20,10 +20,6 @@ matrix:
   include:
     - python: "3.5"
       env: TOXENV=quality
-    - python: "2.7"
```

Comment:
Can you explain why this was removed if you aren't dropping 2.7 in this PR?

Reply:
It's removing just the `pytest-py27` test environment. 2.7 is still in the base matrix (see line 3); the include/exclude here seem to just be modifications on top of the Cartesian cross of `python` and `env`.

It might make sense to drop `python` and `env` and just have a `matrix` that explicitly lists out the python/TOXENV combinations, which I've seen in a different project.

Reply:
You can see that py27 is still being used: https://travis-ci.org/github/edx/opaque-keys/builds/666006599

(I don't really understand why the `python-py27` tox test environment was there to begin with, to be clear.)